### ISSUE TITLE
add tests and CI config

### ISSUE THREAD

**Issue by michaelficarra** [shapesecurity/shift-spec-consumer#1]
Every OSS Shape project must have _some_ tests and use CI. After this is done, we will protect the master branch and gate it on CI.

**Comment by bakkot** [comment]
Added tests and .travis.yml. @michaelficarra, you'll need to enable CI etc.

The tests don't actually use any existing version of shift-spec, but I figure that will be deferred to CI on shift-spec.

Once this is done, it should be published on npm.

**Comment by bakkot** [comment]
Fixed by 4c7754c
